homework6MethodOverloading/CalculateIncomeTaxBasedOnSalary.java

```diff
@@ -0,0 +1,32 @@
+package homework6MethodOverloading;
+
+public class CalculateIncomeTaxBasedOnSalary {
```

Comment:
Calculate Income Tax Based on Salary Using Overloaded Functions (With Special IT Tax Rate)
Write overloaded methods named calculateIncomeTax:
1. One version accepts a gross salary (as a double) and calculates the income tax based on a default tax rate of 20%.
2. Another version accepts a gross salary and a custom tax rate (as a double) to calculate the income tax.
3. A third version accepts a gross salary and a flag indicating whether the person works in the IT sector, applying a special tax rate of 10% for IT employees.